From a853241670badc57fa879175f798fd6e542eb303 Mon Sep 17 00:00:00 2001
From: Jolien Creighton <jolien.creighton@ligo.org>
Date: Mon, 20 Sep 2021 11:23:57 -0700
Subject: [PATCH] make compatible with hdf5 version 1.12 api

---
 lal/lib/support/H5FileIOLowLevel.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lal/lib/support/H5FileIOLowLevel.c b/lal/lib/support/H5FileIOLowLevel.c
index b34bc97d12..5a6a210665 100644
--- a/lal/lib/support/H5FileIOLowLevel.c
+++ b/lal/lib/support/H5FileIOLowLevel.c
@@ -1,6 +1,8 @@
 #include <config.h>
 
 #ifdef HAVE_HDF5
+/* FIXME: should really update interface */
+#define H5_USE_110_API
 #include <hdf5.h>
 #include <hdf5_hl.h>
 #endif
-- 
GitLab

