--- lal.pc.in	2018-11-20 16:34:04.000000000 +0000
+++ lal.pc.in	2019-03-13 17:09:23.000000000 +0000
@@ -12,6 +12,7 @@
 Name: LAL
 Description: LSC Algorithm Library
 Version: @VERSION@
-Requires: gsl, fftw3, fftw3f
-Libs: -L${libdir} -llal @CUDA_LIBS@ @BOINC_LIBS@ @PTHREAD_LIBS@
+Requires.private: gsl, fftw3, fftw3f
+Libs.private: -L${libdir} -llal @CUDA_LIBS@ @BOINC_LIBS@ @PTHREAD_LIBS@
+Libs: -L${libdir} -llal
 Cflags: -I${includedir} @CUDA_CFLAGS@ @BOINC_CFLAGS@ @PTHREAD_CFLAGS@
--- lalsupport.pc.in	2018-11-20 16:34:04.000000000 +0000
+++ lalsupport.pc.in	2019-03-13 17:09:45.000000000 +0000
@@ -13,5 +13,6 @@
 Description: LSC Algorithm Support Library (useful but non-conforming routines)
 Version: @VERSION@
 Requires: lal = @VERSION@
-Libs: -L${libdir} -llalsupport @PTHREAD_LIBS@ @ZLIB_LIBS@
+Libs.private: -L${libdir} -llalsupport @PTHREAD_LIBS@ @ZLIB_LIBS@
+Libs: -L${libdir} -llalsupport
 Cflags: -I${includedir} @PTHREAD_CFLAGS@ @ZLIB_CFLAGS@
