diff --git a/lal/test/tools/ValueTest.c b/lal/test/tools/ValueTest.c
index aee0705eb3..775b59f6b8 100644
--- a/lal/test/tools/ValueTest.c
+++ b/lal/test/tools/ValueTest.c
@@ -35,7 +35,7 @@ static LALDict * create_dict(void)
     dict = XLALCreateDict();
     if (!dict)
         return NULL;
-    err |= XLALDictInsertCHARValue(dict, "CHAR", CHAR_VALUE);
+    //err |= XLALDictInsertCHARValue(dict, "CHAR", CHAR_VALUE);
     err |= XLALDictInsertINT2Value(dict, "INT2", INT2_VALUE);
     err |= XLALDictInsertINT4Value(dict, "INT4", INT4_VALUE);
     err |= XLALDictInsertINT8Value(dict, "INT8", INT8_VALUE);
@@ -61,7 +61,7 @@ static LALList * create_list(void)
         return NULL;
     /* put these in alphabetical order */
     err |= XLALListAddStringValue(list, "BLOB");
-    err |= XLALListAddStringValue(list, "CHAR");
+    //err |= XLALListAddStringValue(list, "CHAR");
     err |= XLALListAddStringValue(list, "COMPLEX16");
     err |= XLALListAddStringValue(list, "COMPLEX8");
     err |= XLALListAddStringValue(list, "INT2");
@@ -140,7 +140,7 @@ int main(void)
         return 1;

     /* make sure the values in the dict are what they should be */
-    TEST(CHAR)
+    //TEST(CHAR)
     TEST(INT2)
     TEST(INT4)
     TEST(INT8)
